content/en/blog/_posts/2020-03-16-kubernetes-the-hard-way.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: blog
+title: SIG Scalability - Learning Kubernetes the Hard Way
```

Comment:
```suggestion
title: Join SIG Scalability and Learn Kubernetes the Hard Way
```

Reply:
Notes on the first few edits: 

- CTA in title, first paragraph, and closing
- Short, descriptive slug (helps with SEO)